bin/rewrite.js

```diff
@@ -8,10 +8,12 @@ const rewriter = require('../lib/rewriter');
  * @param {string} filePath - the path to the file
  */
 function rewriteFile(filePath) {
-  rewriter({
+  const output = rewriter({
     path: filePath,
     source: fs.readFileSync(inputPath, 'utf8'),
```

Comment:
inputPath does not exist but if this was supposed to be filePath then I think it's still wrong, I don't think users will want their original source code over-written. I've re-opened #52